storage/interface.go

```diff
@@ -37,6 +38,8 @@ type Engine interface {
 	GetProvider(moduleName string) Provider
 	// GetSessionDatabase returns the SessionDatabase
 	GetSessionDatabase() SessionDatabase
+
+	SQLDatabase() *gorm.DB
```

Comment:
godoc, other methods are prepended with `Get`. All `Get` or no `Get`?

Reply:
In Go you generally leave out "Get" for "GetXXX", but we should probably do that in one "go". Will change to "GetSQLDatabase()" for the sake of consistency